diff --git a/dist/index.cjs b/dist/index.cjs
index 989e2607b3ecc6c6ad4dd3b20d7e7d38eca92430..d084863c588156a023cb154310e32d366cccc93c 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -115,6 +115,8 @@ const stringCheck = createEslintRule({
     };
     const dict = loadDict(options);
     const code = context.sourceCode.text;
+    const extension = node_path.extname(context.getFilename());
+    const isMarkdown = extension === '.md' || extension === '.mdx';
     const checkText = (node) => {
       const start = node.range[0];
       const end = node.range[1];
@@ -162,7 +164,11 @@ const stringCheck = createEslintRule({
       },
       TemplateElement: (node) => {
         checkText(node);
-      }
+      },
+      Program: (node) => {
+        if (isMarkdown)
+          checkText(node);
+      },
     };
     const templateBodyVisitor = {
       VText(node) {
