diff --git a/node_modules/docusaurus/lib/core/Head.js b/node_modules/docusaurus/lib/core/Head.js
index f8a8548..8146d90 100644
--- a/node_modules/docusaurus/lib/core/Head.js
+++ b/node_modules/docusaurus/lib/core/Head.js
@@ -22,7 +22,7 @@ class Head extends React.Component {
     // Use user-provided themeUrl if it exists, else construct one from version and theme.
     const highlightThemeURL = highlight.themeUrl
       ? highlight.themeUrl
-      : `//cdnjs.cloudflare.com/ajax/libs/highlight.js/${highlight.version}/styles/${highlight.theme}.min.css`;
+      : `//cdn.jsdelivr.net/npm/highlight.js@${highlight.version}/styles/${highlight.theme}.min.css`;
 
     // ensure the siteUrl variable ends with a single slash
     const siteUrl = `${(
@@ -147,7 +147,7 @@ class Head extends React.Component {
           )}
 
         {this.props.config.scrollToTop && (
-          <script src="https://unpkg.com/vanilla-back-to-top@7.1.14/dist/vanilla-back-to-top.min.js" />
+          <script src="https://cdn.jsdelivr.net/npm/vanilla-back-to-top@7.1.14/dist/vanilla-back-to-top.min.js" />
         )}
         {this.props.config.scrollToTop && (
           <script
@@ -168,19 +168,19 @@ class Head extends React.Component {
             }}
           />
         )}
-        <script src={`${this.props.config.baseUrl}js/scrollSpy.js`} />
+        <script src={`${this.props.config.cdnUrl}js/scrollSpy.js`} />
         {this.props.config.usePrism && (
           <link
             rel="stylesheet"
-            href={`${this.props.config.baseUrl}css/prism.css`}
+            href={`${this.props.config.cdnUrl}css/prism.css`}
           />
         )}
         {/* Site defined code. Keep these at the end to avoid overriding. */}
         <link
           rel="stylesheet"
-          href={`${this.props.config.baseUrl}css/main.css`}
+          href={`${this.props.config.cdnUrl}css/main.css`}
         />
-        <script src={`${this.props.config.baseUrl}js/codetabs.js`} />
+        <script src={`${this.props.config.cdnUrl}js/codetabs.js`} />
       </head>
     );
   }
