diff --git a/node_modules/docusaurus/lib/core/Head.js b/node_modules/docusaurus/lib/core/Head.js
index f8a8548..87b030a 100644
--- a/node_modules/docusaurus/lib/core/Head.js
+++ b/node_modules/docusaurus/lib/core/Head.js
@@ -22,7 +22,7 @@ class Head extends React.Component {
     // Use user-provided themeUrl if it exists, else construct one from version and theme.
     const highlightThemeURL = highlight.themeUrl
       ? highlight.themeUrl
-      : `//cdnjs.cloudflare.com/ajax/libs/highlight.js/${highlight.version}/styles/${highlight.theme}.min.css`;
+      : `//cdn.jsdelivr.net/npm/highlight.js@${highlight.version}/styles/${highlight.theme}.min.css`;
 
     // ensure the siteUrl variable ends with a single slash
     const siteUrl = `${(
@@ -69,7 +69,7 @@ class Head extends React.Component {
         )}
         <link
           rel="shortcut icon"
-          href={this.props.config.baseUrl + this.props.config.favicon}
+          href={this.props.config.favicon}
         />
         {this.props.config.algolia && (
           <link
@@ -147,7 +147,7 @@ class Head extends React.Component {
           )}
 
         {this.props.config.scrollToTop && (
-          <script src="https://unpkg.com/vanilla-back-to-top@7.1.14/dist/vanilla-back-to-top.min.js" />
+          <script src="https://cdn.jsdelivr.net/npm/vanilla-back-to-top@7.1.14/dist/vanilla-back-to-top.min.js" />
         )}
         {this.props.config.scrollToTop && (
           <script
@@ -168,19 +168,19 @@ class Head extends React.Component {
             }}
           />
         )}
-        <script src={`${this.props.config.baseUrl}js/scrollSpy.js`} />
+        <script src={`${this.props.config.cdnUrl}js/scrollSpy.js`} />
         {this.props.config.usePrism && (
           <link
             rel="stylesheet"
-            href={`${this.props.config.baseUrl}css/prism.css`}
+            href={`${this.props.config.cdnUrl}css/prism.css`}
           />
         )}
         {/* Site defined code. Keep these at the end to avoid overriding. */}
         <link
           rel="stylesheet"
-          href={`${this.props.config.baseUrl}css/main.css`}
+          href={`${this.props.config.cdnUrl}css/main.css`}
         />
-        <script src={`${this.props.config.baseUrl}js/codetabs.js`} />
+        <script src={`${this.props.config.cdnUrl}js/codetabs.js`} />
       </head>
     );
   }
diff --git a/node_modules/docusaurus/lib/core/nav/HeaderNav.js b/node_modules/docusaurus/lib/core/nav/HeaderNav.js
index 0b3d468..b8160ce 100644
--- a/node_modules/docusaurus/lib/core/nav/HeaderNav.js
+++ b/node_modules/docusaurus/lib/core/nav/HeaderNav.js
@@ -309,7 +309,7 @@ class HeaderNav extends React.Component {
               {siteConfig.headerIcon && (
                 <img
                   className="logo"
-                  src={this.props.baseUrl + siteConfig.headerIcon}
+                  src={siteConfig.headerIcon}
                   alt={siteConfig.title}
                 />
               )}
diff --git a/node_modules/docusaurus/lib/server/docs.js b/node_modules/docusaurus/lib/server/docs.js
index 3602ded..b1024f2 100644
--- a/node_modules/docusaurus/lib/server/docs.js
+++ b/node_modules/docusaurus/lib/server/docs.js
@@ -102,6 +102,7 @@ function mdToHtmlify(oldContent, mdToHtml, metadata, siteConfig) {
   return content;
 }
 
+const docsAssetsLocation = 'https://cdn.jsdelivr.net/gh/reactnativecn/react-native-website@gh-pages/docs';
 function getMarkup(rawContent, mdToHtml, metadata, siteConfig) {
   // generate table of contents
   let content = insertTOC(rawContent);
@@ -110,9 +111,6 @@ function getMarkup(rawContent, mdToHtml, metadata, siteConfig) {
   content = mdToHtmlify(content, mdToHtml, metadata, siteConfig);
 
   // replace any relative links to static assets (not in fenced code blocks) to absolute links
-  const docsAssetsLocation = siteConfig.docsUrl
-    ? `${siteConfig.baseUrl}${siteConfig.docsUrl}`
-    : siteConfig.baseUrl.substring(0, siteConfig.baseUrl.length - 1);
   content = replaceAssetsLink(content, docsAssetsLocation);
 
   const DocsLayout = require('../core/DocsLayout.js');
