diff --git a/node_modules/react-native/types/public/ReactNativeTypes.d.ts b/node_modules/react-native/types/public/ReactNativeTypes.d.ts
index 38e00e1..3acd235 100644
--- a/node_modules/react-native/types/public/ReactNativeTypes.d.ts
+++ b/node_modules/react-native/types/public/ReactNativeTypes.d.ts
@@ -86,9 +86,11 @@ export interface NativeMethods {
    * _Can also be called with a relativeNativeNodeHandle but is deprecated._
    */
   measureLayout(
-    relativeToNativeComponentRef: HostComponent<unknown> | number,
+    relativeToNativeComponentRef:
+      | React.ElementRef<HostComponent<unknown>>
+      | number,
     onSuccess: MeasureLayoutOnSuccessCallback,
-    onFail: () => void /* currently unused */,
+    onFail?: () => void,
   ): void;
 
   /**
